### ISSUE TITLE
React Toggle problems

### ISSUE THREAD

**Issue by pearofducks** [fabric-ds/issues#113]
- DeadToggle has `name="test"` and probably shouldn't (it's at least confusing and seems like an oversight)
- Keyboard nav doesn't work out of the box (with fClickable at least), and should
- `noVisibleLabel` prop messes up the outline on the checkbox ([slack thread](https://sch-chat.slack.com/archives/C01GYKPJVFT/p1667224414142549))

**Comment by martinetruls** [comment]
It seems that if the <Clickable/> component would get another prop for adding a common "name", would help a lot to make the "next" radio in the group become selected.

Here is an example where I changed the DOM to a common name "hei":

https://user-images.githubusercontent.com/16682340/197702852-350e4efe-089d-40e0-a4dc-fb39642af49b.mov

![Skjermbilde 2022-10-25 kl  08 48 00](https://user-images.githubusercontent.com/16682340/197702866-0f241d28-8be7-4b82-bdb5-96839e37a019.png)
